src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -1684,38 +1784,6 @@ test_cluster_time_updated_during_handshake ()
    mongoc_uri_destroy (uri);
 }
 
-/* returns the last time the topology completed a full scan. */
-static int64_t
-_get_last_scan (mongoc_client_t *client)
```

Comment:
This test required some changing. _get_last_scan is no longer a thing for multi-threaded since servers are scanned independently.